app/pulp/app/settings.py

```diff
@@ -165,18 +165,22 @@
             'simple': {'format': 'pulp: %(name)s:%(levelname)s: %(message)s'},
         },
         'handlers': {
-            'stream': {
+            'pulp': {
                 'class': 'logging.StreamHandler',
                 'formatter': 'simple',
             },
         },
         'loggers': {
-            'pulp.app': {
-                'handlers': ['stream'],
+            '': {
+                # The root logger
+                'handlers': ["pulp"],
+            },
+            'pulp': {
+                'handlers': ["pulp"],
                 'level': 'INFO',
             },
             'django': {
-                'handlers': ['stream'],
+                'handlers': ['pulp'],
```

Comment:
Similar question here, do we even want a `django` logger that restricts the `level` or should we just use a root logger?